PROTO_tests/tests/stats_test.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+import arkouda as ak
+
+
+class TestStats:
+    def setup_class(cls):
```

Comment:
this should have an `@classmethod` decorator

Reply:
Added